targets/ar71xx-generic/profiles.mk

```diff
@@ -1,5 +1,12 @@
 # List of hardware profiles
 
+## GL-iNet
+
+# GL-iNet 1.0 ; BB is the 8MB version, works on 16M too
+$(eval $(call GluonProfile,GLINET))
+$(eval $(call GluonModel,GLINET,gl-inet-v1-squashfs,gl-inet-v1.0))
```

Comment:
The image filename `gl-inet-v1.0` looks wrong. As far as I can tell the correct name is `gl-connect-inet-v1` (the name is essential for the autoupdater).